File: apex-arquivos/Lógica/aula_05/js/pessoa.js
// Variáveis = Cria-se com letra minúscula no início da palavra
// Classes = Cria-se com letra MAIÚSCULA no início da palavra

let Pessoa = {
    nome: 'Diego',
    idade: 33,

    apresentacao: function () {
        // return this.nome.concat(' - ').concat(this.idade);
        // OU:
        return `${this.nome} - ${this.idade}`;
    }
}

document.write(Pessoa.apresentacao());

File: apex-arquivos/C# Fundamentos/aula_05/exercicio/Pessoas/Pessoa.cs
namespace exercicio.Pessoas
{
    public class Pessoa
    {
        private string Nome { get ; set; }
        private int Idade { get ; set; }

        public Pessoa(string nome, int idade)
        {
            this.Nome = nome;
            this.Idade = idade;
        }

        public string ObterDescricao()
        {
            return $"Nome: {this.Nome} - Idade: {this.Idade}";
        }
    }
}